app/code/Magento/Config/Controller/Adminhtml/System/AbstractConfig.php

```diff
@@ -58,6 +67,10 @@ public function __construct(
      */
     public function dispatch(\Magento\Framework\App\RequestInterface $request)
     {
+        if (!$this->_sessionsManager->getCurrentSession()->isLoggedInStatus()) {
+            $this->messageManager->addSuccessMessage(__('You were logged out due to inactivity.'));
```

Comment:
Please use the default message, which already has translations: "Your current session has been expired."